python/lsst/ip/diffim/imageDecorrelation.py

```diff
@@ -24,7 +24,6 @@
 #
 
 import numpy as np
-import scipy.fftpack
```

Comment:
Please also remove the lines
```
from future import standard_library
standard_library.install_aliases()
```
from near the top of the file -- these are no longer needed and make a hash of flake8 linting.